### PR TITLE
added Gitea support

### PR DESCRIPTION
**Pull request by paspiz85**

### CODE REVIEW

**Comment by julian70400** [comment]
Hello, what's up of this merge ?

**Comment by dregad** [comment]
There was an earlier attempt at implementing this, as discussed in #255 - see the work-in-progress by @fokkersim https://github.com/fokkersim/SourceGitea but as far as I can see there has been no progress since 2022.

I'll review this PR and merge it in a new release as time allows (hopefully by end of this week). It will remain community-supported as I do not use Gitea myself.

**Comment by julian70400** [comment]
Thank you so much.

For information, I tryed to install it, with Mantis 2.26, but got this error : _control source integration 2.5.0_ 

Not sure about what it mean,.

**Comment by julian70400** [comment]
For information : https://github.com/fokkersim/SourceGitea/issues/1

**Comment by dregad** [comment]
@julian70400 if the plugin this PR is to become the "official" version, then you're going to add confusion by opening issues in @fokkersim's work-in-progress version...

**Comment by julian70400** [comment]
@dregad Sorry, I removed this.

File: SourceGitea/SourceGitea.php [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+if ( false === include_once( config_get( 'plugin_path' ) . 'Source/MantisSourceGitBasePlugin.class.php' ) ) {
+	return;
+}
+
+require_once( config_get( 'core_path' ) . 'url_api.php' );
+require_once( config_get( 'core_path' ) . 'json_api.php' );
+
+class SourceGiteaPlugin extends MantisSourceGitBasePlugin {
+
+	const PLUGIN_VERSION = '1.0.1';
```
[the remark this answers is not in the document]
Comment (paspiz85) [reply SourceGitea/SourceGitea.php]: I don't remember but I changed to 1.0.0

**Comment by dregad** [comment]
Merged in 8c268daa4b7d912e2d1c7656eef04800026efb20

**Comment by dregad** [comment]
Thanks for your contribution @paspiz85 - v2.8.0 including the new SourceGitea plugin is out.

**Pull request closed without merging**